Shared/Samples/Animate 3D graphic/Animate3DGraphicView.swift

```diff
@@ -93,15 +96,18 @@ struct Animate3DGraphicView: View {
         .task {
             await model.monitorCameraController()
         }
-        .onDisappear {
-            model.animation.stop()
-        }
     }
     
     /// The list containing the mission settings.
     private var missionSettings: some View {
         List {
             Section("Mission") {
+                VStack {
+                    StatRow("Progress", value: model.animation.progress.formatted(.rounded))
```

Comment:
The `StatRow("Progress")` view could use some top padding.

Mac Catalyst:
<img width="316" alt="Screenshot 2024-03-29 at 1 06 06 PM" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/c5ec5d82-5ca9-4cde-ac71-b63a38dbcf25">